ui/src/journals/components/JournalTitlesListModal.tsx

```diff
@@ -14,6 +14,7 @@ export const JournalTitlesListModal = ({
     visible={modalVisible}
     onCancel={onModalVisibilityChange}
     footer={null}
+    maskStyle={{ width: '100vw', overflow: 'hidden' }}
```

Comment:
I'm not sure this is the right solution, why this is happening? We have other modals and they work properly.

Reply:
@karolina-siemieniuk-morawska ☝️